src/redisearch_rs/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [workspace]
 members = [
+    "entrypoint/*",
```

Comment:
Perhaps `c_entrypoint` is a better name, thinking about it now.

Reply:
Done https://github.com/RediSearch/RediSearch/pull/6254/commits/1091bc3a89200a0890ce40b7172c9bf69cc5bb3c